src/AspNet.Security.OAuth.Validation/Events/AuthenticationFailedContext.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.AspNetCore.Http;
 using System;
 
-namespace AspNet.Security.OAuth.Validation.Events
```

Comment:
I was just following what was there in the ASP.NET team and ASOS, but yeah, usage is a bit easier for consumers as well as they don't need the sub-namespace and it's really only used for organization of the code.